v1/send.md

```diff
@@ -100,3 +100,27 @@ order: 1
     }
 }
 ```
+
+## Inline Images
+
+### Sample Request Code
```

Comment:
This and the template example below should be H4's to match the rest of the document.